src/main/java/com/autotune/common/k8sObjects/Metric.java

```diff
@@ -33,9 +33,11 @@ public final class Metric {
     private final String query;
     private final String datasource;
     private final String valueType;
+    private String kubernetesObject;
     private final LinkedHashMap<String, EMMetricResult> trialSummaryResult = new LinkedHashMap<String, EMMetricResult>();
     private EMMetricResult emMetricResult;
     private final LinkedHashMap<String, LinkedHashMap<String, LinkedHashMap<Integer, EMMetricResult>>> cycleDataMap = new LinkedHashMap<String, LinkedHashMap<String, LinkedHashMap<Integer, EMMetricResult>>>();
+    private AggregationFunctions aggregationFunctions;
```

Comment:
This needs to be a hashmap as there will be multiple aggregationFunctions

Reply:
Updated